cogito/protocol_test.go

```diff
@@ -209,6 +209,7 @@ func TestPutParamsPrintLogRedaction(t *testing.T) {
 		ChatMessage:     "stecchino",
 		ChatMessageFile: "dir/msg.txt",
 		GChatWebHook:    "sensitive-gchat-webhook",
+		Sinks:           []string{"gchat", "github"},
```

Comment:
you can as well use some random names here. To be consistent with `johnny` and `stecchino`

Reply:
That one should be real, I could also remove it because it defaults to all supported sinks.

Reply:
Can you explain why? I am missing this point. Is `func (putter *ProdPutter) Sinks() ([]Sinker, error)` somehow called before?

My understanding only String method is called in this test.

Reply:
Ah it is called: `params.Sinks`